npm install gradient is not working

I don't know why but it seems as if it is not installing correctly. This is what I'm doing:

    $ npm install color
> ...
> color@0.7.3 node_modules/color
> ├── color-string@0.2.1
> └── color-convert@0.5.2

    $ npm install gradient
> ...
> gradient@0.2.0 node_modules/gradient

    $ node

    > var c = require('color');
> undefined

    > var Gradient = require('gradient');
>Error: Cannot find module '/color'
    at Function.resolve (/.../node_modules/gradient/gradient.js:41:15)
    at require (/.../node_modules/gradient/gradient.js:3:28)
    at /.../node_modules/gradient/gradient.js:2027:9
    at Object.<anonymous> (/.../node_modules/gradient/gradient.js:2027:28)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
>

--- a/.npmignore
+++ b/.npmignore
@@ -1,3 +1,3 @@
-./gradient.js
-./gradient-min.js
+gradient.js
+gradient-min.js
 

--- a/.npmignore
+++ b/.npmignore
@@ -1,3 +1,3 @@
 gradient.js
-gradient-min.js
+gradient.min.js
 

--- a/package.json
+++ b/package.json
@@ -25,6 +25,6 @@
         "browserify": "~1.1.0"
       , "uglify-js": "1.0.x"
     }
-  , "main": "gradient"
+  , "main": "lib/gradient.js"
   , "engines": [ "node >= 0.4"]
 }

